### DISCUSSION TITLE
[Adonis 4.1] Unhandled promise rejection. Cannot found the promise who is causing this warning.

### DISCUSSION THREAD

**Discussion by wemersonrv** [adonisjs/core#1488] [Help] [answered]
Hello,

I'm deploying an app, that runs perfectly on my local environment. no errors. But when i publish it gives me `WARNING: Adonis has detected an unhandled promise rejection`.

The issue is that i cannot detect what exact promise, because it don't tell nothing more. So, there is a way to make a more detailed log to catch it?

**Comment by thetutlage** [comment, marked as answer]
Run your app as `node --trace-warnings server.js` and it will show the complete stack trace

**Reply by wemersonrv** [reply]
Tx my friend. Problem solved. Your the best!
